File: AtCoder/asa/2020/asa1011/b.py
e = list(map(int, input().split()))
b = int(input())
l = list(map(int, input().split()))
n = 6
cnt = 0
bonus = 0
for i in range(n):
    if(l[i] in e): cnt += 1
    else:
        if(l[i] == b): bonus = 1

if(cnt == 6): print(1)
else:
    if(cnt == 5):
        if(bonus == 1): print(2)
        else: print(3)
    
    else:
        if(cnt < 3): print(0)
        else: print(8-cnt)

File: AtCoder/asa/2020/asa1011/b.cpp
#include <bits/stdc++.h>
#define _GLIBCXX_DEBUG
#define rep(i, n) for(int i = 0; i < (n); ++i)
#define rep1(i, n) for(int i = 1; i < (n); ++i)
using namespace std;
typedef long long ll;

int main(){
    cin.tie(nullptr);
    ios::sync_with_stdio(false);

    int n = 6;
    vector<int> e(n), l(n);
    rep(i, n) cin >> e[i];
    int b;
    cin >> b;
    rep(i, n) cin >> l[i];

    int cnt = 0;
    int bonus = 0;
    rep(i, n){
        if(e[i] == l[i]) ++cnt;
        else{
            if()
        }
    }

    return 0;
}